### PR TITLE
Release 1.2.9

### PR DESCRIPTION
**Pull request by stephen-cox**
* Updated GitHub workflow

* Coding standards fixes

### CODE REVIEW

File: localgov_base.libraries.yml [hunk 1]
```diff
@@ -90,7 +90,7 @@ page-title-block:
       css/components/page-title-block.css: {}
 
 service-landing-page:
-  css:
+  css:) ? TRUE : FALSE;
```
Comment (finnlewis) [inline localgov_base.libraries.yml]: @stephen-cox is this line meant to be like this?

Comment (stephen-cox) [reply localgov_base.libraries.yml]: @finnlewis Bloody IDE pasting stuff it shouldn't. Fixed now

**Review by finnlewis — APPROVED** [review]
Nice one!

File: localgov_base.libraries.yml (hunk 1 above)
Comment (finnlewis) [reply localgov_base.libraries.yml]: That's what the second pair of eyes are for!

**Pull request merged**